packages/discord.js-utilities/src/lib/type-guards.ts

```diff
@@ -1,24 +1,24 @@
 import { isNullish, Nullish } from '@sapphire/utilities';
-import type { APIGuildMember, APIInteractionDataResolvedGuildMember, APIInteractionGuildMember, APIMessage } from 'discord-api-types/v9';
+import { APIGuildMember, APIInteractionDataResolvedGuildMember, APIInteractionGuildMember, APIMessage, ChannelType } from 'discord-api-types/v10';
 import {
+	BaseInteraction,
```

Comment:
```suggestion
import { 
        type APIGuildMember,
        type APIInteractionDataResolvedGuildMember,
        type APIInteractionGuildMember,
        type APIMessage,
        ChannelType,
	BaseInteraction,
```

Reply:
And organized imports after 
(VSCode organizer imports puts types last)